src/pages/nonprofit/issues/[id].tsx

```diff
@@ -1,9 +1,218 @@
+            borderColor="border"
+            borderRadius="lg"
+            backgroundColor="surface"
+            align="stretch"
+            paddingY={3}
+            justifyContent="space-between"
+            paddingX={6}
+            //spacing={10}
+          >
+            <HStack spacing={2}>
+              <Text
+                fontSize="sm"
+                fontWeight="bold"
+                padding="5px 10px"
+                color={
+                  issue.status == IssueStatus.CLOSED ? "#79c099" : "#cf7f21"
+                }
+                bgColor={
+                  issue.status == IssueStatus.CLOSED ? "#ecf6f0" : "#f0eae3"
+                }
+                borderRadius="md"
+              >
+                {issue.status == IssueStatus.CLOSED ? "Complete" : "Pending"}
+              </Text>
+              <HStack spacing={1}>
+                <Text fontSize="sm" color="#657788">
+                  Issued {dateToMMDDYYYY(issue.createdAt)}
+                </Text>
+                {issue.status == IssueStatus.CLOSED && (
+                  <Text fontSize="sm" color="#657788">
+                    , Closed {dateToMMDDYYYY(issue.finishedAt)}
+                  </Text>
+                )}
+              </HStack>
+            </HStack>
+            {issue.status == IssueStatus.PENDING && (
+              <Link href={urls.pages.nonprofit.issues.index} passHref={true}>
+                <Button variant="secondary" fontSize="sm" onClick={closeIssue}>
+                  Cancel Request
+                </Button>
+              </Link>
+            )}
+          </HStack>
+
+          <VStack
+            maxW="1000px"
+            border="1px solid"
+            borderColor="border"
+            borderRadius="lg"
+            backgroundColor="surface"
+            align="stretch"
+          >
+            <VStack align="stretch" p="20px" spacing={5}>
+              <VStack align="stretch">
+                <Text fontSize="sm" fontWeight="bold">
+                  Issue #
+                </Text>
+                <Text fontSize="sm">{issue._id}</Text>
+              </VStack>
+              <VStack align="stretch">
+                <Text fontSize="sm" fontWeight="bold">
+                  Issue Type
+                </Text>
+                <Text fontSize="sm">{issue.type}</Text>
+              </VStack>
+              <VStack align="stretch">
+                <Text fontSize="sm" fontWeight="bold">
+                  Description
+                </Text>
+                <Text fontSize="sm">{issue.description}</Text>
+              </VStack>
+              <VStack align="stretch">
+                <Text fontSize="sm" fontWeight="bold">
+                  Screenshots
+                </Text>
+                <HStack>
+                  {issue.images.map((image, index) => (
+                    <VStack key={index} align="start">
+                      {/* <Image boxSize="100px" src={image.blobPath} /> */}
+
+                      <Image
```

Comment:
Very easy mistake to make, but this should be Image from Next.js, not from Chakra.  To still use onClick, you can wrap the image in a Box and add the onClick on the Box.  This applies to all images.